bitnami/mariadb/templates/primary/statefulset.yaml

```diff
@@ -60,7 +60,7 @@ spec:
       tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.primary.tolerations "context" $) | nindent 8 }}
       {{- end }}
       {{- if .Values.priorityClassName }}
```

Comment:
It seems like the `if .Values.priorityClassName` line should also be updated.

Reply:
Thanks for the heads up